docs/home/api-info/webhooks.mdx

```diff
@@ -0,0 +1,130 @@
+to your desired URL making another request to the same endpoint.
+
+## Format
+
+Webhook requests contain the relevant information on the body of the HTTP request.
+
+There are three types of messages you can expect to receive:
+
+- [`ping`](#the-ping-message): validation of the webhook connection between Metriport and your app;
+- [user data](#devices-api): health information grouped by user (Devices API only).
+- [patient data](#medical-api): patient document information (Medical API only).
+
+In general, upon successful receiving of these messages, it's expected that your app responds with a `200` HTTP
+status code (OK).
+
+There's no need to include anything on the response body.
+
+### The `ping` message
+
+This is a simple message to validate access to your app's webhook endpoint. Your app should
+accept a `POST` request with this body...
+
+```json
+{
+  "ping": "<random-sequence>"
+}
+```
+
+...and respond to this request by sending back the `<random-sequence>` as below:
+
+```json
+{
+  "pong": "<random-sequence>"
+}
+```
+
+You can check the [webhook mock server available on our repository](https://github.com/metriport/metriport/blob/master/utils/src/mock-webhook.ts)
+for a simple implementation of this message.
+
+### Devices API messages
+
+When using the Devices API, Metriport will send Webhook messages containing user data to your app from
+connected Providers, as soon as the data becomes available.
+
+You can see Webhook details specific to the Devices API on [this page](/devices-api/more-info/webhooks).
+
+### Medical API messages
+
+When using the Medical API, Metriport will send Webhook messages containing patient document data to your app, as soon
+as the data becomes available after initiating a document query.
+
+You can see Webhook details specific to the Medical API on [this page](/medical-api/more-info/webhooks).
+
+## Retries
+
+Sometimes, due to temporary network distruption (or whatever other reason), a Webhook payload may fail to be delivered
+to your app. In these cases, Metriport will store the failed requests, and then you can then manually retry them - using
+either the dashboard, or the API.
+
+### Retry Using the Dashboard
+
+On the [Developers page](https://dash.metriport.com/developers) in the dashboard, you're able to see the count of
+Webhook requests currently processing, and a count of outstanding ones that failed:
+
+<img className="h-100" src="/images/dash-webhook-retry.png" />
+
+To retry failed requests, simply click the `Retry` button, and those requests will be sent to your app again.
+
+### Retry Using the API
+
+1. Using the [Get Webhook Status endpoint](/home/api-reference/settings/get-webhook), check to see if there are any failed requests;
+1. Then, use the [Retry Webhook requests endpoint](/home/api-reference/settings/retry-webhook), to kick off the retry.
+
+<Warning>
+  Currently, the Metriport doesn't implement automatic retries - let us know if this is something
```

Comment:
nit: `Currently, the Metriport doesn't...` > `Currently, the Metriport API doesn't...` | `Currently, Metriport doesn't...`

Reply:
🕵️